c_glib/arrow-glib/field.cpp

```diff
@@ -177,14 +185,13 @@ garrow_field_get_name(GArrowField *field)
  * garrow_field_get_data_type:
  * @field: A #GArrowField.
  *
- * Returns: (transfer full): The data type of the field.
+ * Returns: (transfer none): The data type of the field.
```

Comment:
This breaks backward compatibility for C API users (not for GObject Introspection based bindings users).
Normally, it's not desired. But we can accept this for this case because users who use this function may not exist.

Reply:
I see.